### PR TITLE
[MU3] Fix #322759: Inputing Ottava Bassa via menu or shortcut shows it above staff

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Resolves: https://musescore.org/en/node/322759

PR for master in #8516 (merged already)

### CODE REVIEW

File: mscore/menus.cpp [hunk 1]
```diff
@@ -1414,6 +1414,7 @@ PalettePanel* MuseScore::newLinesPalettePanel()
       ottava = new Ottava(gscore);
       ottava->setOttavaType(OttavaType::OTTAVA_22MB);
       ottava->setLen(w);
+      ottava->setPlacement(Placement::BELOW);
```
Comment (Jojo-Schmitz) [inline mscore/menus.cpp:1417]: May not be needed, certainily not for thwe issue at hane but found this to be pretty inconsistent, at least.

File: libmscore/edit.cpp [hunk 2]
```diff
@@ -1435,6 +1435,10 @@ void Score::cmdAddOttava(OttavaType type)
                        cr2 = cr1;
                   Ottava* ottava = new Ottava(this);
                   ottava->setOttavaType(type);
+                  if (type == OttavaType::OTTAVA_8VB/* || type == OttavaType::OTTAVA_15MB || type == OttavaType::OTTAVA_22MB*/) {
```
Comment (Jojo-Schmitz) [inline libmscore/edit.cpp:1438]: No need to check for `OttavaType::OTTAVA_15MB` and `OttavaType::OTTAVA_22MB`, as there's no shortcut for those

File: libmscore/edit.cpp [hunk 3]
```diff
@@ -1453,7 +1457,10 @@ void Score::cmdAddOttava(OttavaType type)
 
             Ottava* ottava = new Ottava(this);
             ottava->setOttavaType(type);
-
+            if (type == OttavaType::OTTAVA_8VB/* || type == OttavaType::OTTAVA_15MB || type == OttavaType::OTTAVA_22MB*/) {
```
Comment (Jojo-Schmitz) [inline libmscore/edit.cpp:1460]: same

**Comment by wizofaus** [comment]
What's branch 3.x used for? I thought there weren't going to be any more 3.x versions released (only the backend for doing online conversions etc.)

**Comment by Jojo-Schmitz** [comment]
See #9000

**Pull request closed without merging**